docs/comparison-of-builders.md

```diff
@@ -0,0 +1,221 @@
+# Comparison of DiCy to other LaTeX Builders
+
+There are many different automatic and semi-automatic builders for LaTeX-centric
+documents. The [Compilation Topic][] at CTAN provides a current list of some of
+those specifically designed for TeX or LaTeX documents. In addition there are
+also generic document processing tools such as [pandoc][].
+
+In order to explain the intention of and capabilities of DiCy this document
+compares DiCy to two of the most popular builders, [arara][] and [latexmk][].
+
+## arara
+
+[arara][] is a rule based builder which relies on directives given in the source
```

Comment:
IMHO, link to external page only at the first appearance in the document is enough.